mentat/command/commands/screenshot.py

```diff
@@ -39,9 +41,11 @@ async def apply(self, *args: str) -> None:
                 color="green",
             )
         except ScreenshotException:
+            return  # Screenshot manager will print the error to stream.
+        except WebDriverException:
             stream.send(
-                'No browser open. Run "/screenshot path" with a url or local file',
-                color="red",
+                "Error taking screenshot. Please run with a valid url or local path.",
```

Comment:
I would prefer if this except was inside vision_manager.screenshot; that way we don't have to rewrite this except if we decide to use .screenshot again somewhere.